libraries/botbuilder-expression/src/buildInFunction.ts

```diff
@@ -1068,6 +1305,48 @@ export class BuiltInFunctions {
             BuiltInFunctions.StringTransform(ExpressionType.ToLower, (args: ReadonlyArray<any>) => String(args[0]).toLowerCase()),
             BuiltInFunctions.StringTransform(ExpressionType.ToUpper, (args: ReadonlyArray<any>) => String(args[0]).toUpperCase()),
             BuiltInFunctions.StringTransform(ExpressionType.Trim, (args: ReadonlyArray<any>) => String(args[0]).trim()),
+            new ExpressionEvaluator(
+                ExpressionType.StartsWith,
+                BuiltInFunctions.Apply((args: ReadonlyArray<any>) => args[0].startsWith(args[1]), BuiltInFunctions.VerifyString),
+                ReturnType.Boolean,
+                (expression: Expression): void => BuiltInFunctions.ValidateArityAndAnyType(expression, 2, 2, ReturnType.String),
+            ),
+            new ExpressionEvaluator(
+                ExpressionType.EndsWith,
+                BuiltInFunctions.Apply((args: ReadonlyArray<any>) => args[0].endsWith(args[1]), BuiltInFunctions.VerifyString),
+                ReturnType.Boolean,
+                (expression: Expression): void => BuiltInFunctions.ValidateArityAndAnyType(expression, 2, 2, ReturnType.String),
+            ),
+            new ExpressionEvaluator(
+                ExpressionType.CountWord,
+                BuiltInFunctions.Apply((args: ReadonlyArray<any>) => args[0].trim().split(/\s+/).length, BuiltInFunctions.VerifyString),
+                ReturnType.Number,
+                BuiltInFunctions.ValidateUnaryString
+            ),
+            new ExpressionEvaluator(
+                ExpressionType.AddOrdinal,
+                BuiltInFunctions.Apply((args: ReadonlyArray<any>) => this.AddOrdinal(args[0]), BuiltInFunctions.VerifyInteger),
+                ReturnType.Number,
+                (expression: Expression): void => BuiltInFunctions.ValidateArityAndAnyType(expression, 1, 1, ReturnType.Number),
+            ),
+            new ExpressionEvaluator(
+                ExpressionType.Guid,
+                BuiltInFunctions.Apply(() => BuiltInFunctions.newGuid()),
+                ReturnType.String,
+                (expression: Expression): void => BuiltInFunctions.ValidateArityAndAnyType(expression, 0, 0, ReturnType.String),
```

Comment:
there is no need to write `ReturnType.String`

Reply:
Right as there is no param. Fixed